repo/packages/D/datastax-ops/200/resource.json

```diff
@@ -0,0 +1,62 @@
+{
+  "assets": {
+    "container": {
+      "docker": {
+        "opscenter": "dspn/10517a0c8e9b11e79ef5600308a463ab-2:6.1.2"
+      }
+    },
+    "uris": {
+      "jre-tar-gz": "https://downloads.mesosphere.com/java/server-jre-8u162-linux-x64.tar.gz",
+      "libmesos-bundle-tar-gz": "https://downloads.mesosphere.com/libmesos-bundle/libmesos-bundle-master-28f8827.tar.gz",
```

Comment:
libmesos-bundle seems to be the old? why not libmesos-bundle-1.11.0.tar.gz?

Reply:
@benclarkwood

Reply:
The externals weren't updated yet.

This is a well tested bundle that is what's currently running everywhere that is GA.